### PR TITLE
Improvement for issue #44 by adding paging to the event view table

### PR DESCRIPTION
**Pull request by davidpbauer**
Refactor attendancerecordprovider to fix getdata to allow for paging. Update
the event view table to show 25 records at a time.

### CODE REVIEW

**Review by ottenhoff — APPROVED** [review]
Awesome thanks

File: tool/src/java/org/sakaiproject/attendance/tool/pages/EventView.java [hunk 1]
```diff
@@ -293,6 +295,9 @@ public Component getLazyLoadComponent(String s){
                 };
                 item.add(dataPanelSlow);
             }
-        });
+        };
+        recordDataView.setItemsPerPage(25);
```
Comment (ottenhoff) [inline tool/src/java/org/sakaiproject/attendance/tool/pages/EventView.java:299]: Can we do serverConfigurationService.getInt("attendance.pager.items", 25)

Comment (davidpbauer) [reply tool/src/java/org/sakaiproject/attendance/tool/pages/EventView.java]: Great suggestion. I've made it configurable but default 25

**Comment by ottenhoff** [comment]
Here is a comment from Laura C at ND:

_Since this huge course site is actually a supersection of 40 much smaller sections which meet at different times, we would expect the grader to filter for their section first.

So the workflow to be:
(1) TA or Instructor clicks on the specific attendance item title. Take Attendance page starts to load with all student names.
(3) Grader interrupts page load by filtering for a specific section using View dropdown. (Not sure how to know if Sakai recognizes this choice.)
(4) Section loads, grader starts grading.

Unfortunately, when the grader clicks on the attendance item right now, the entire page of students starts to load. Not sure at what point that process is affected by grader choosing the View filter, if at all._

**Comment by ottenhoff** [comment]
Your PR here would certainly improve this behavior and make it easier to move between sections.

**Comment by davidpbauer** [comment]
Yes, feel free to merge. Looks like I don't have permission to merge

**Pull request merged**